Documentation/packaging.md

```diff
@@ -0,0 +1,65 @@
+# packaging rkt
+
+## Build-time dependencies
+
+#### Basic build-time dependencies
+
+- bash
+- go
+- gofmt
+- file
+- git
+- gpg
+
+#### Additional build-time dependencies for the coreos flavor
+
+wget, gpg, mktemp, md5sum, cpio, gzip, unsquashfs, sort
+
+#### Additional build-time dependencies for the src flavor
+
+intltoolize, libtoolize and all systemd dependencies
+
+## Run-time dependencies
+
+#### Basic run-time dependencies
+
+Linux configured with [suitable options](Documentation/hacking.md#run-time-requirements)
+
+#### Additional run-time dependencies for the host flavor
+
+- systemd >= v222 with systemctl, systemd-shutdown, systemd, systemd-journald.
+- bash
+
+## Ownership and permissions of rkt directories
+
+In general, subdirectories of `/var/lib/rkt` should be created with the same ownership and permissions as done by `rkt install`, see [directory list](https://github.com/coreos/rkt/blob/master/rkt/install.go#L44).
+
+The package should create a group `rkt` and `/var/lib/rkt` should belong to group `rkt` with the `setgid` bit set (`chmod g+s`)
+
+TODO: add better explanation
+
+## Offline builds
+
+By default, rkt compilation will download a CoreOS PXE image from internet and extract some binaries such as `systemd-nspawn` and `bash`. However, some packaging environments dont allow internet access during the build.
+
+To solve this, you can download the CoreOS PXE image before starting the build process and use the following options:
+
+```
+./configure --with-coreos-local-pxe-image-path=/data/coreos_production_pxe_image.cpio.gz --with-coreos-local-pxe-image-systemd-version=v222
+```
+
+## Code bundle
+
+Most Linux distributions dont allow to use pre-build binaries or to reuse copy of code that is already otherwise packaged. systemd falls in this category, as Debian and Fedora already package systemd and rkt needs systemd.
+
+- [Debian Policy Manual, section 4.13 Convenience copies of code](https://www.debian.org/doc/debian-policy/ch-source.html#s-embeddedfiles)
```

Comment:
@jonboulle In https://github.com/coreos/rkt/issues/1594#issuecomment-147608080 you suggest to deprecate the `host` flavor. But at the moment it is the only flavor suitable for Debian/Fedora because of their packaging policies (offline builds, not bundling code from other packages, not picking binaries/libs from the build host). Did you have an idea of how to do things differently?